src/modules/hunting/kubelet.py

```diff
@@ -207,14 +207,15 @@ def test_port_forward(self):
 
         # executes one command and returns output
         def test_run_container(self):
+            fake_container_name = "fake-container"
             run_url = self.path + self.Handlers.RUN.value.format(
                 podNamespace=self.pod["namespace"],
                 podID=self.pod["name"],
-                containerName=self.pod["container"],
+                containerName=fake_container_name,
                 cmd = ""
             )
-            status_code = requests.post(run_url, allow_redirects=False, verify=False).status_code 
-            return (status_code != 404 and status_code != 401)
+            output = requests.post(run_url, allow_redirects=False, verify=False).text
```

Comment:
I don't think we should be doing *any* POST requests in a passive hunter, so I think this needs to be moved to an active hunter. Wdyt?

Reply:
basically it's the same as accessing /pods endpoint.
Take a look at the link I posted, because we are making up a container name, we will never pass on to the function that executes something.

Reply:
Agreed it's unlikely that anyone will have a container called "fake_container"... but still, I am not sure this is in the spirit of passive hunters being "read only", as it's attempting to make a state-changing operation, albeit one that it expects to fail. @mcherny wdyt?

Reply:
I think it's the best check we can do to ensure the run handler is accessible, the alternative is to remove that check and make it only available for the active Hunter

Reply:
@lizrice @danielsagi I tend to agree that even if the POST won't do anything (because the container don't exist) it's still semantically 'write' operation and potentially can cause side effects (e.x. error logs). Hence should not be used by passive hunters.